tests/e2e/utils/activate-amp-and-set-mode.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * The allow list of AMP modes.
+ *
+ */
+export const allowedAmpModes = {
+	standard: 'standard',
+	transitional: 'transitional',
+	reader: 'disabled',
+};
+
+/**
+ * Activate AMP and set it to the correct mode.
+ *
+ * @param {string} mode The mode to set AMP to. Possible value of standard, transitional or reader.
+ */
+export const activateAmpAndSetMode = async ( mode ) => {
```

Comment:
```suggestion
export const activateAMPAndSetMode = async ( mode ) => {
```
Also, maybe `activateAMPWithMode` as an alternate name?